deploy/compose/compose-walkers.yaml

```diff
@@ -2,7 +2,7 @@ version: '3'
 services:
   vexination-walker:
     image: $TRUST_IMAGE:${TRUST_VERSION:?TRUST_VERSION is required}
-    command: vexination walker --devmode --source https://www.redhat.com/.well-known/csaf/provider-metadata.json -3 --storage-endpoint http://minio:9000
+    command: vexination walker --devmode --workers 3 --source https://www.redhat.com/.well-known/csaf/provider-metadata.json -3 --storage-endpoint http://minio:9000
```

Comment:
Technically valid and the right thing to do. However, as I implemented and tested it (with more than 3 workers though), I right away got blacklisted by the CDN (Akamai, Cloudflare, ?).

If you tested it with 3, and 3 works, then it's ok.